bin/postserve

```diff
@@ -17,7 +17,7 @@ Usage:
 Options:
   -l --layer=<layer>    If set, limit tile generation to just this layer (could be multiple)
   -x --exclude-layers   If set, uses all layers except the ones listed with -l (-l is required)
-  -s --serve=<port>     Return this hostname as tileserver URL in metadata  [default: localhost]
+  -s --serve=<url>      Return this URL as tileserver URL in metadata  [default: http://localhost:8090]
```

Comment:
Need to substitute default port for the `--port` param. Since docopt auto-sets this value, you can simply do a 
`url = args['--serve'].replace('<port>', args['--port'])`
using this change in doc:

```suggestion
  -s --serve=<url>      Return this URL as tileserver URL in metadata  [default: http://localhost:<port>]
```

Reply:
Good catch. done.